src/network/networkd-util.h

```diff
@@ -7,10 +7,12 @@
 
 typedef enum AddressFamilyBoolean {
         /* This is a bitmask, though it usually doesn't feel that way! */
-        ADDRESS_FAMILY_NO   = 0,
-        ADDRESS_FAMILY_IPV4 = 1 << 0,
-        ADDRESS_FAMILY_IPV6 = 1 << 1,
-        ADDRESS_FAMILY_YES  = ADDRESS_FAMILY_IPV4 | ADDRESS_FAMILY_IPV6,
+        ADDRESS_FAMILY_NO             = 0,
+        ADDRESS_FAMILY_IPV4           = 1 << 0,
+        ADDRESS_FAMILY_IPV6           = 1 << 1,
+        ADDRESS_FAMILY_YES            = ADDRESS_FAMILY_IPV4 | ADDRESS_FAMILY_IPV6,
+        ADDRESS_FAMILY_FALLBACK       = ADDRESS_FAMILY_IPV4 | ADDRESS_FAMILY_IPV6 | 1 << 2,
+        ADDRESS_FAMILY_FALLBACK_IPV4  = ADDRESS_FAMILY_IPV4 | 1 << 3,
```

Comment:
It is not necessary to include `ADDRESS_FAMILY_IPV4` in `ADDRESS_FAMILY_FALLBACK`. How about the following:
```
ADDRESS_FAMILY_FALLBACK_IPV4 = 1 << 2,
ADDRESS_FAMILY_FALLBACK = ADDRESS_FAMILY_FALLBACK_IPV4 | ADDRESS_FAMILY_IPV6,
```